api/src/main/java/uk/gov/dhsc/htbhf/claimant/message/processor/DetermineEntitlementMessageProcessor.java

```diff
@@ -66,48 +59,26 @@ public MessageStatus processMessage(Message message) {
         PaymentCycle currentPaymentCycle = messageContext.getCurrentPaymentCycle();
         PaymentCycle previousPaymentCycle = messageContext.getPreviousPaymentCycle();
 
-        EligibilityResponse eligibilityResponse = eligibilityService.determineEligibilityForExistingClaimant(claimant);
-
-        if (eligibilityResponse.getEligibilityStatus() == ELIGIBLE) {
-            messageQueueClient.sendMessage(buildMakePaymentMessagePayload(currentPaymentCycle), MAKE_PAYMENT);
-        }
+        EligibilityAndEntitlement eligibility = eligibilityService.determineEligibilityForExistingClaimant(
+                claimant,
+                currentPaymentCycle.getCycleStartDate(),
+                previousPaymentCycle);
 
         //TODO HTBHF-1296 - update ClaimStatus from ACTIVE to PENDING_EXPIRY if Claimant is no longer eligible.
+        updateAndSaveCurrentPaymentCycle(currentPaymentCycle, eligibility);
 
-        PaymentCycleVoucherEntitlement voucherEntitlement = determineVoucherEntitlement(
-                currentPaymentCycle,
-                previousPaymentCycle,
-                eligibilityResponse,
-                claimant);
-        updateAndSaveCurrentPaymentCycle(currentPaymentCycle, eligibilityResponse, voucherEntitlement);
-
-        return COMPLETED;
-    }
-
-    private PaymentCycleVoucherEntitlement determineVoucherEntitlement(PaymentCycle currentPaymentCycle,
-                                                                       PaymentCycle previousPaymentCycle,
-                                                                       EligibilityResponse eligibilityResponse,
-                                                                       Claimant claimant) {
-
-        if (eligibilityResponse.getEligibilityStatus() != ELIGIBLE) {
-            return null;
+        if (eligibility.getEligibilityStatus() == ELIGIBLE) {
+            messageQueueClient.sendMessage(buildMakePaymentMessagePayload(currentPaymentCycle), MAKE_PAYMENT);
         }
 
-        Optional<LocalDate> expectedDeliveryDate = Optional.ofNullable(claimant.getExpectedDeliveryDate());
-        List<LocalDate> dateOfBirthOfChildren = eligibilityResponse.getDateOfBirthOfChildren();
-        return cycleEntitlementCalculator.calculateEntitlement(
-                expectedDeliveryDate,
-                dateOfBirthOfChildren,
-                currentPaymentCycle.getCycleStartDate(),
-                previousPaymentCycle.getVoucherEntitlement());
+        return COMPLETED;
     }
 
     private void updateAndSaveCurrentPaymentCycle(PaymentCycle currentPaymentCycle,
-                                                  EligibilityResponse eligibilityResponse,
-                                                  PaymentCycleVoucherEntitlement paymentCycleVoucherEntitlement) {
+                                                  EligibilityAndEntitlement eligibility) {
```

Comment:
I think this should be renamed to `eligibilityAndEntitlement` as it looks really odd below when you call `eligibility.getVoucherEntitlement()`.